bootstrapping_decision_making.md

```diff
@@ -28,3 +28,5 @@ This framework has the following characteristics:
 
 - By gradually increasing the size of the decision making body one person at a time, the power implicit in this election process is gradually distributed over a larger group of people.
 - At each stage where the new decision making body is considering adding a new person, they can also choose to stop electing additional members. This allows the group to decide on the overall size in an organic, gradual manner.
+- In the spirit of Jupyter always seeking to develop multi-stakeholder governance, this bootstrapping process should try to maximize the participation of individuals from multiple organizations. Specifically, they should when possible alternate across organizations to avoid all initial members of the decision making body belonging to a single organization.
```

Comment:
In the spirit of Jupyter governance being inclusive, this bootstrapping process should maximize the participation of individuals from multiple organizations and backgrounds. Specifically, the bootstrapping process should avoid all initial members of the decision-making body belonging to a single organization or project. Further, the bootstrapping process should aim for inclusion.

Reply:
See our comment above, we have modified the text to address this...